docs/2021-05-27-debug-feature-tests.md

```diff
@@ -0,0 +1,14 @@
+# Debug feature tests
+
+Add a `binding.pry` into the context of a test that you want to investigate.
+
+Copy the name of the test (relative path) of the test in question, run `rspect` in your terminal with the `CHROME` flag.
+
+For example, I want to run the `clients_searching_sorting_and_filtering_spec` spec:
+
+```bash
+# In the root of the vita-min project
+CHROME=y rspec ./spec/features/hub/clients_searching_sorting_and_filtering_spec.rb
```

Comment:
Trivial, but consider removing `./` - the following does the job and is 2 bytes shorter & 1% clearer.

`CHROME=y rspec spec/features/hub/clients_searching_sorting_and_filtering_spec.rb`